frontend/packages/ux-editor/src/components/config/editModal/EditOptions/OptionTabs/EditTab/OptionListEditor/LibraryOptionsEditor/LibraryOptionsEditor.tsx

```diff
@@ -0,0 +1,104 @@
+import type { Option } from 'app-shared/types/Option';
+import React, { createRef, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import {
+  StudioCodeListEditor,
+  StudioModal,
+  StudioAlert,
+  StudioParagraph,
+  StudioButton,
+} from '@studio/components';
+import type { CodeListEditorTexts } from '@studio/components';
+import { PencilIcon, TrashIcon } from '@studio/icons';
+import { usePreviewContext } from 'app-development/contexts/PreviewContext';
+import { useStudioEnvironmentParams } from 'app-shared/hooks/useStudioEnvironmentParams';
+import { useUpdateOptionListMutation } from 'app-shared/hooks/mutations';
+import { useOptionListEditorTexts } from '@altinn/ux-editor/components/config/editModal/EditOptions/OptionTabs/hooks';
+import type { OptionListEditorProps } from '../OptionListEditor';
+import classes from './LibraryOptionsEditor.module.css';
+
+type LibraryOptionsEditorProps = {
+  optionsList: Option[];
+  handleDelete: () => void;
+} & Pick<OptionListEditorProps, 'optionsId' | 'component'>;
+
+export function LibraryOptionsEditor({
+  component,
+  optionsId,
+  optionsList,
+  handleDelete,
+}: LibraryOptionsEditorProps): React.ReactNode {
+  const { t } = useTranslation();
+  const { org, app } = useStudioEnvironmentParams();
+  const { doReloadPreview } = usePreviewContext();
+  const { mutate: updateOptionList } = useUpdateOptionListMutation(org, app);
+  const [localOptionList, setLocalOptionList] = useState<Option[]>(optionsList);
```

Comment:
Is this state necessary? 
And nitpicking, but it should be `[localOptionsList, setLocalOptionsList]` as in the variable in the initial state 🙈

Reply:
After testing, likely no it is no longer needed